_inc/client/plans/single-product-backup.jsx

```diff
@@ -35,6 +42,83 @@ export function PlanPriceDisplay( { backupPlanPrices, currencySymbol } ) {
 	);
 }
 
+class SingleProductBackupBody extends React.Component {
+	static propTypes = {
+		backupPlanPrices: PropTypes.object,
+		currencySymbol: PropTypes.string,
+		setSelectedBackupType: PropTypes.func,
+		selectedBackupType: PropTypes.string,
+		upgradeLinks: PropTypes.object,
+	};
+
+	constructor() {
+		super();
+
+		this.handleSelectedBackupTypeChange = this.handleSelectedBackupTypeChange.bind( this );
+	}
+
+	handleSelectedBackupTypeChange( event ) {
+		this.props.setSelectedBackupType( event.target.value );
+	}
+
+	render() {
+		const { currencySymbol, backupPlanPrices, selectedBackupType, upgradeLinks } = this.props;
+
+		const upgradeTitles = {
+			'real-time': __( 'Upgrade to Real-Time Backups' ),
+			daily: __( 'Upgrade to Daily Backups' ),
+		};
+
+		const backupOptions = [
+			{
+				type: 'daily',
+				name: __( 'Daily Backups' ),
+				price: backupPlanPrices.jetpack_backup_daily.yearly,
+			},
+			{
+				type: 'real-time',
+				name: __( 'Real-Time Backups' ),
+				price: backupPlanPrices.jetpack_backup_realtime.yearly,
+			},
+		];
+
+		return (
+			<div className="plans-section__body">
+				<p>
+					{ __(
+						'Always-on backups ensure you never lose your site. Choose from real-time or daily backups. {{a}}Which one do I need?{{/a}}',
+						{
+							components: {
+								a: <ExternalLink href="https://jetpack.com/upgrade/backup/" icon iconSize={ 12 } />,
+							},
+						}
+					) }
+				</p>
+
+				<h4>{ __( 'Backup options:' ) }</h4>
+				<div className="single-product-backup__radio-buttons-container">
+					{ backupOptions.map( option => (
+						<PlanRadioButton
```

Comment:
Do we need a `key` attribute here to avoid warnings?

Reply:
Yes, fixed.

Reply:
Done in 237176521064985cb78c28a85be2f7e1ba6f7559. Marking this as resolved.